web/private/scripts/db_cleanup.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Don't ever sanitize the database on the live environment. Doing so would
+// destroy the canonical version of the data.
+if (defined('PANTHEON_ENVIRONMENT') && (PANTHEON_ENVIRONMENT !== 'live')) {
+
+	$webform_query = "truncate table webform_submission; truncate table webform_submission_data; truncate table webform_submission_log;";
+	$watchdog_query = "truncate table watchdog;";
+
+	// Delete webform submission data
+	echo "Deleting webform submissions...\n";
+	passthru('drush sql-query "' . $webform_query . '"');
+	echo "Webform submission deletion complete.\n";
+
+	// Delete log data
+	echo "Deleting log entries...\n";
+	passthru('drush sql-query "' . $watchdog_query . '"');
```

Comment:
@jookyg, sorry to come up with another issue... but I just thought of another issue with dropping the watchdog logs on all all our multidevs and dev. Just today, Richard needed to look at the watchdog log on dev to see what was happening during a build failure. I don't think we should delete that table. We are only keeping 10,000 log entries now, so it is a lot smaller already.